scripts/generate-integration-pages.ts

```diff
@@ -31,6 +31,10 @@ const prettyCategoryDescription: Record<string, unknown> = {
 	adapter: 'SSR adapter to deploy your Astro project',
 	other: 'integration in your Astro project',
 };
+
+// Add pages titles that should be marked as `i18nReady: false` as part of this array.
+const translationBlacklist: string[] = [];
```

Comment:
I’d suggest using “blocklist” instead of “blacklist” or a use-case specific name like `i18nNotReadyIntegrations` or something. We can also follow the same pattern as for `#deprecatedIntegrations` setting this inside `IntegrationsPagesBuilder`.

Maybe we want to block translation of Markdoc for now given it’s still very experimental?

Reply:
Fair points, I'll make these changes after arriving from college 🙌